gdscript-customization.el

```diff
@@ -138,5 +148,21 @@ directory path containing the file `index.html'."
   :type 'string
   :group 'gdscript)
 
+(defcustom gdscript-docs-online-api-url "https://docs.godotengine.org/en/stable/classes/index.html?#godot-api"

```

Comment:
Do we need these variables to be customizable?

The only case I can think of is browsing docs for specific versions of Godot, like "latest" or 3.1. In which case we shouldn't expose the full string templates as a public variable, but rather an enum of available versions.

I'd generally keep values like https://docs.godotengine.org/en/stable/classes/class_%s.html#%s private as the functions rely on the template to work.

Reply:
I tend to like anything that appears more than once to be a customisable (as an aside I wouldn't keep all customiszables in a separate file bur rather in the files they are used but that's another issue). That said, I'm happy to adhere to your guidelines and remove them.